src/components/track-changes/TrackChangesPanel.tsx

```diff
@@ -25,6 +26,21 @@ import { SnapshotsDropdown } from '../inspector/SnapshotsDropdown'
 import { SortByDropdown } from './SortByDropdown'
 import { SuggestionList } from './suggestion-list/SuggestionList'
 
```

Comment:
I'm not a big fan of doing the scrolling here. We should be able to achieve the same result with https://prosemirror.net/docs/ref/#view.EditorProps.handleScrollToSelection while keeping the logic contained within the body-editor.

Reply:
I will take care of this in LEAN-3694. Thanks for your input!